DapperDoodle/ICommandExecutor.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace DapperDoodle
+{
+    public interface ICommandExecutor
+    {
+        void Execute(Command command);
+
```

Comment:
Can remove spaces here between the methods just for consistency with other interfaces